silx/gui/plot/actions/io.py

```diff
@@ -220,7 +224,8 @@ def _getAxesLabels(self, item):
         ylabel = item.getYLabel() or self.plot.getYAxis().getLabel()
         return xlabel, ylabel
 
-    def _selectWriteableOutputGroup(self, filename):
+    @staticmethod
```

Comment:
It cannot be static, it's using `self`

Reply:
OK, I got why you want it static, then self needs to be replaced by SaveAction.

Reply:
Yes sorry! Forgot to change it